.github/workflows/dockerimage.yml

```diff
@@ -0,0 +1,48 @@
+name: Docker Image CI
+on:
+  push:
+    branches: 
+      - master
+      - rc
```

Comment:
Neat! Would it be possible to do a pattern here as well? E.g. `release-*`. That's the branch name of the next release candidate